pennylane_lightning/src/Apply.cpp

```diff
@@ -68,6 +68,20 @@ void Pennylane::constructAndApplyOperation(
     gate->applyKernel(state, internalIndices, externalIndices, inverse);
 }
 
+void Pennylane::applyGateGenerator(
+    StateVector& state,
+    unique_ptr<AbstractGate> gate,
+    const vector<unsigned int>& opWires,
+    const unsigned int qubits
```

Comment:
Do we need `inverse` here? For the adjoint diff method, I don't think so.

Reply:
I don't think so either. I think it's fine not having it for now.